gapic-generator-ads/.gitignore

```diff
@@ -1,4 +1,4 @@
-Gemfile.lock
+!Gemfile.lock
```

Comment:
Can this just be omitted?

Reply:
The root .gitignore still forbids Gemfile.lock -s since I am not sure whether we want the root's Gemfile.lock to be included. 
From that we have to allow Gemfile.lock -s in all subfolders

Reply:
fixed this in a different way